examples/goldilocks_models/find_models/initial_guess.h

```diff
@@ -16,7 +16,8 @@ std::string SetInitialGuessByInterpolation(const std::string& directory,
                                            int iter, int sample,
                                            const TasksGenerator* task_gen,
                                            const Task& task, const RomData& rom,
-                                           bool use_database, int robot);
+                                           bool use_database, int robot,
+                                           bool is_test=false);
```

Comment:
Since we can choose the value for `theta` in the test file, we shouldn't add a flag `is_test` here